docs/Displaying a User Interface/Adaptive Interface Capabilities/index.md

```diff
@@ -1,34 +1,34 @@
+The following properties are deprecated on SDL @![iOS]iOS 6.4!@@![android, javaSE, javaEE]Android 4.10!@@![javascript]JavaScript 1.0!@ because as of RPC v6.0 they are deprecated. However, these properties will still be filled with information. When connected on RPC <6.0, the information will be exactly the same as what is returned in the `RegisterAppInterfaceResponse` and `SetDisplayLayoutResponse`. However, if connected on RPC >6.0, the information will be converted from the newer-style display information, which means that some information will not be available.
 
 | Parameters | Description |
 | ---------- | ----------- |
-| @![iOS]displayCapabilities!@@![android, javaSE, javaEE]SystemCapabilityType.DISPLAY!@ | Information about the HMI display. This includes information about available templates, whether or not graphics are supported, and a list of all text fields and the max number of characters allowed in each text field. |
-| @![iOS]buttonCapabilities!@@![android, javaSE, javaEE]SystemCapabilityType.BUTTON!@ | A list of available buttons and whether the buttons support long, short and up-down presses. |
-| @![iOS]softButtonCapabilities!@@![android, javaSE, javaEE]SystemCapabilityType.SOFTBUTTON!@ | A list of available soft buttons and whether the button support images. Also, information about whether the button supports long, short and up-down presses. |
-| @![iOS]presetBankCapabilities!@@![android, javaSE, javaEE]SystemCapabilityType.PRESET_BANK!@ | If returned, the platform supports custom on-screen presets. |
+| @![iOS]displayCapabilities!@@![android, javaSE, javaEE, javascript]SystemCapabilityType.DISPLAY!@ | Information about the HMI display. This includes information about available templates, whether or not graphics are supported, and a list of all text fields and the max number of characters allowed in each text field. |
+| @![iOS]buttonCapabilities!@@![android, javaSE, javaEE, javascript]SystemCapabilityType.BUTTON!@ | A list of available buttons and whether the buttons support long, short and up-down presses. |
+| @![iOS]softButtonCapabilities!@@![android, javaSE, javaEE, javascript]SystemCapabilityType.SOFTBUTTON!@ | A list of available soft buttons and whether the button support images. Also, information about whether the button supports long, short and up-down presses. |
+| @![iOS]presetBankCapabilities!@@![android, javaSE, javaEE, javascript]SystemCapabilityType.PRESET_BANK!@ | If returned, the platform supports custom on-screen presets. |
 
```

Comment:
Since none of the deprecated ones were added to the JS library then none of these ones exist in JS